cfgov/cfgov/wsgi.py

```diff
@@ -10,6 +10,24 @@
 import os
 
 
+import dotenv
+
+this_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+def initialize_environment():
+    environment_candidates = [
+        os.path.join(this_dir, '../.env'),
+        os.path.join(this_dir, '../../.env'),
+        os.path.join(os.getcwd(), '.env')
+    ]
+
+    for candidate in environment_candidates:
+        if os.path.exists(candidate):
```

Comment:
What about when none of the candidates are found?  Is that OK?  In a Docker world, I'm guessing you wouldn't use a `.env` file at all, and would just pass in envvars right to the container?

Perhaps a log statement saying which `.env` file is being used, or none at all?

Reply:
now, instead of searching for candidates, it tries to read ../../env. If it isn't there, the [dotenv library will issue a warning](https://github.com/jpadilla/django-dotenv/blob/master/dotenv.py#L64)